cuchem/benchmark/scripts/benchmark.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS smiles (
```

Comment:
@rilango It appears the PR contains changes for more than just customer bug fixes, KMeans workflow change, and RAPIDS version bump. These changes are related to new benchmarking code. Were they intentionally included? Decision is yours if we include them -- in general, best not to release untested new features, but also they are unlikely to matter for customers since this is benchmark focused.

Reply:
This PR was suppose to be just RAPIDS 21.08. But while working on upgrading to RAPIDS 21.08, I had made changes to Benchmark code as well as .